.gitignore

```diff
@@ -14,10 +14,8 @@ out/*
 *.rar
 
 # Assignments
-asn*
-## A1
-src/main/java/Country.java
-src/main/java/CountryCatalogue.java
+src/main/java/A1Solution/
```

Comment:
Prefer to abide by [package naming conventions](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html).